07/main07.py

```diff
@@ -0,0 +1,166 @@
+# 1. Створити телефонну книгу, яка матиме наступні команди:
+#
+# stats: кількість записів
+#
+# add: додати запис
+#
+# delete <name>: видалити запис за іменем (ключем)
+#
+# list: список всіх імен в книзі
+#
+# show <name>: детальна інформація по імені
+#
+# Записи не мають повторюватися, заборонено перезаписувати записи, тільки видаляти і додавати заново.
+#
+
+# importing some necessary libs here
+import os
+import time
+# lib import finished
+
+phone_dic = {
+    'entry0': None,
+    'entry1': 10,
+    'entry2': 20,
+    'entry3': 30,
+    'entry4': 40,
+    'entry5': 50,
+}
+
+def dic_count_items(c_var_i):
+    counter = 0
+    for key0,value0 in c_var_i.items():
+        counter += 1
+    return counter
+
+
+# decided to do it in cycle, omg here we go
+while True:
+    os.system('cls')
+    inp = input('\n'
+                'Welcome to the test phonebook.\n'
+                'Here is the list of available commands:\n'
+                'stats      - shows you the number of entries stored\n'
+                'add        - adds a new entry\n'
+                'delete     - deletes an entry\n'
+                'list       - shows you all entries in the phonebook\n'
+                'show       - tells you details about entry\n'
+                '\n'
+                'Enter command: ')
+
+    inp_l = str(inp.lower())
+
+    # list ---------------------------
+    if inp_l == 'list':
+        os.system('cls')
+        print('\n')
+        for k, v in phone_dic.items():
+            print(k, v)
+        list_inp = input('\n'
+                        'Enter anything to start over: ')
+        if list_inp is not '':
```

Comment:
Here you have 2 possible conditions to continue your code, in other words, you always continue your code. In this case, maybe you don't need extra input from user. You can replace all this part (lines 59 - 64) with only one `continue` line

Reply:
Good point. I just thought that I could use a user input as a workaround to let the user read through entries until user decides to leave the page and go back to the main menu. I understand that I've overcomplicated things at this point but I wanted to make the output of "list" command on a separate page (which of course I understand was not that necessary).

Reply:
Let me clarify my point: nothing will change if you remove these lines. For both conditions, you run `continue` command. In this case, you can remove the condition and just leave `continue` here

Reply:
Ohhh, I've tested it and now I understand what you meant. It is fixed.